cmd/mdatagen/loader.go

```diff
@@ -122,7 +122,7 @@ type metric struct {
 	Sum *sum `mapstructure:"sum,omitempty"`
 	// Gauge stores metadata for gauge metric type
 	Gauge *gauge `mapstructure:"gauge,omitempty"`
-	// Gauge stores metadata for gauge metric type
+	// Histogram stores metadata for histogram metric type
```

Comment:
Split to its own PR

Reply:
PTAL https://github.com/open-telemetry/opentelemetry-collector/pull/11057